### PR TITLE
Add support for using more than 1 sender thread.

### PR DESCRIPTION
**Pull request by baldersheim**
@hmusum or @bratseth PR
@vekterli FYI

### CODE REVIEW

File: vespaclient-core/src/main/java/com/yahoo/feedhandler/VespaFeedHandler.java [hunk 1]
```diff
@@ -100,6 +165,9 @@ public HttpResponse handle(HttpRequest request, RouteMetricSet.ProgressCallback
             }
 
             sender.done();
+            if (feedAccess instanceof ThreadedFeedAccess) {
+                ((ThreadedFeedAccess)feedAccess).close();
+            }
```
Comment (hmusum) [inline vespaclient-core/src/main/java/com/yahoo/feedhandler/VespaFeedHandler.java]: Perhaps the SimpleFeedAccess interface could have a close method instead?

**Review by hmusum — APPROVED** [review]
One comment, LGTM

**Review by bratseth — APPROVED** [review]

File: vespaclient-core/src/main/java/com/yahoo/feedhandler/VespaFeedHandler.java (hunk 1 above)
Comment (bratseth) [reply vespaclient-core/src/main/java/com/yahoo/feedhandler/VespaFeedHandler.java]: And if you are making changes I suggest making ThreadedFeedAccess a top-level class.

Comment (baldersheim) [reply vespaclient-core/src/main/java/com/yahoo/feedhandler/VespaFeedHandler.java]: What does top-level class imply ?

Comment (bratseth) [reply vespaclient-core/src/main/java/com/yahoo/feedhandler/VespaFeedHandler.java]: Not nested inside another class.

**Pull request merged**